share/spack/docker/spack_fedora/modules.yaml

```diff
@@ -0,0 +1,47 @@
+# -------------------------------------------------------------------------
+# This is the default configuration for Spack's module file generation.
+#
+# Settings here are versioned with Spack and are intended to provide
+# sensible defaults out of the box. Spack maintainers should edit this
+# file to keep it current.
+#
+# Users can override these settings by editing the following files.
+#
+# Per-spack-instance settings (overrides defaults):
+#   $SPACK_ROOT/etc/spack/modules.yaml
+#
+# Per-user settings (overrides default and site settings):
+#   ~/.spack/modules.yaml
+# -------------------------------------------------------------------------
+modules:
```

Comment:
if you want to overwrite the default within user space, it should be enough to write the section
```yaml
modules:
  enable:
     - lmod
```
in this file.

I would also check this file in only once in the directory above for all docker recipes.